core/modules/compare/store/getters.ts

```diff
@@ -6,7 +6,8 @@ const getters: GetterTree<CompareState, RootState> = {
   isEmpty: state => state.items.length === 0,
   isOnCompare: state => product => state.items.some(p => p.sku === product.sku),
   isCompareLoaded: state => state.loaded,
-  getCompareProductsCount: state => state.items.length
+  getCompareProductsCount: state => state.items.length,
+  getItems: state => state.items
```

Comment:
```suggestion
  getCompareItems: state => state.items
```
easier to track unique getter name `getItems` is more likely to be used in different contexts around application